app/scripts/background.js

```diff
@@ -906,6 +909,18 @@ export function setupController(
           }
         });
       }
+      if (
```

Comment:
In phishing communication, the new tab opened could be a phishing page. But in this case, if we open a new page, that page could send us a cookie so that both scenarios could happen simultaneously, right?!

Reply:
Yes, I guess if any of the allowed origins ever did want to do a eth_requestAccounts call, then this would be needed.